containers/Ideas/ExplorePage.js

```diff
@@ -27,6 +28,18 @@ export const ExplorePage = (props) => {
       }).catch(error => console.warn(error))
   }
 
+  const fetchAuthenticatedUser = () => {
+    AsyncStorage.getItem('authToken')
+      .then((token) => API.getAuthenticatedUser(token))
+      .then((response) => {
+        if(response.statusCode === 200) {
+          setUser(response.data)
+        } else {
+          return Promise.reject(new Error('fetchAuthenticatedUser returned ' + response.statusCode))
+        }
+      }).catch(error => console.warn(error))
```

Comment:
little nit-picky here, in another PR we decided to put `.catch` on the new line because of:

copying the entire comment here (because linking lead to something else):

> i think it is more consistent to have it on the same indentation as the `.then` above
> 
> ```
> fetching()
>   .then()
>   .then()
>   .catch()
> ```
> 
> vs.
> 
> ```
> fetching()
>   .then()
>   .then().catch()
> ```